src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+input::placeholder {
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 400;
```

Comment:
The font with this font-weight value isn't added to @font-face, consider fixing it